lib/anyway/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Anyway # :nodoc:
-  VERSION = "2.0.2"
+  VERSION = "2.0.3"
```

Comment:
Please, do not update versions in PRs. You never know when this PR will be merged and which version would be the next one)

Reply:
Nice point. Fixed.